CI Update

v-alje opened the pull request:
Build.Reason:Manual by Alma Jenks
 Build.Url:https://apidrop.visualstudio.com/Content%20CI/_build/results?buildId=337496&view=results
 source_repo.branch:master
 source_repo.url:https://apidrop.visualstudio.com/_git/binaries

Commit:
CI Update
 Build.Reason:Manual by Alma Jenks
 Build.Url:https://apidrop.visualstudio.com/Content%20CI/_build/results?buildId=337496&view=results
 source_repo.branch:master
 source_repo.url:https://apidrop.visualstudio.com/_git/binaries

Commit:
CI Update
 Build.Reason:Manual by Alma Jenks
 Build.Url:https://apidrop.visualstudio.com/Content%20CI/_build/results?buildId=337502&view=results
 source_repo.branch:master
 source_repo.url:https://apidrop.visualstudio.com/_git/binaries

ghogen committed:
Fix cref link

ghogen committed:
Fix cref link

ghogen committed:
remove link that gave warning

ghogen committed:
remove link that gave warning

ghogen committed:
seealso tag -> see tag

ghogen committed:
seealso tag -> see tag

ghogen committed:
delink reference to internal type

ghogen committed:
delink reference to internal type

ghogen committed:
delink ref to private field

ghogen committed:
delink ref to private field

gewarren commented on msbuild-api-docs/xml/Microsoft.Build.BuildEngine/Project.xml:
```diff
@@ -1121,7 +1121,7 @@
       <Docs>
         <summary>Gets a collection of the item groups specified with the ItemGroup element in the project.</summary>
-        <value>A <see cref="T:Microsoft.Build.BuildEngine.BuildItemGroupCollection" /> containing the item groups specified in the project.</value>
+        <value>A <see cref="T:Microsoft.Build.BuildEngine.BuildItemGroupCollection" /> containing the item groups specifiied in the project.</value>
```
Can you run it again without the XML files? It looks like most/all of the documentation changes are undesirable.

v-alje replied:
The XML for 4.8.1 has been removed and the CI job is being re-run.

v-alje replied:
Sorry @ghogen, this will reset some of your changes.

gewarren reviewed (changes requested):
Does this include updates from XamlBuildTask.dll too?

ghogen committed:
remove remarks section for Field (removes build warning)

v-alje commented:
> Does this include updates from XamlBuildTask.dll too?

Yes.

Commit:
CI Update
 Build.Reason:Manual by Alma Jenks
 Build.Url:https://apidrop.visualstudio.com/Content%20CI/_build/results?buildId=337509&view=results
 source_repo.branch:master
 source_repo.url:https://apidrop.visualstudio.com/_git/binaries

gewarren commented on msbuild-api-docs/xml/PackageInformation/msbuild-15-netcore.json:
```diff
@@ -0,0 +1 @@
+{"msbuild-15-netcore":{"Microsoft.Build":{"Name":"Microsoft.Build","Version":"15.9.20","Feed":"https://api.nuget.org/v3/index.json"},"Microsoft.Build.Framework":{"Name":"Microsoft.Build.Framework","Version":"15.9.20","Feed":"https://api.nuget.org/v3/index.json"},"Microsoft.Build.Tasks.Core":{"Name":"Microsoft.Build.Tasks.Core","Version":"15.9.20","Feed":"https://api.nuget.org/v3/index.json"},"Mic ...
```
Are these JSON files expected?

v-alje replied:
Yes. These provide the information for listing the NuGet package on the pages generated from NuGet packages.

gewarren commented on msbuild-api-docs/xml/Microsoft.Build.Experimental/OutOfProcServerNode.xml:
```diff
@@ -0,0 +1,72 @@
+<Type Name="OutOfProcServerNode" FullName="Microsoft.Build.Experimental.OutOfProcServerNode">
```
These experimental types are new. Is that expected?

ghogen replied:
We should add the Experimental namespace to the filter

v-alje replied:
The namespaces have been added to the _filter.xml.

v-alje committed:
Adding Experimental Namespaces to _filter

ghogen committed:
Delink broken cref link

ghogen reviewed (approved):
We can safely ignore the remaining warnings. There is one type that references the Experimental namespace, that we filtered out, which will be a broken link, I'll notify the product team.

ghogen commented:
@v-alje  Ready to merge and publish, thanks!
